pkg/operator/podidentity/gcppodidentitywebhook.go

```diff
@@ -0,0 +1,34 @@
+package podidentity
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+const gcpFolder = "v4.1.0/gcp-pod-identity-webhook"
```

Comment:
Minor: where does this version number come from? OpenShift 4.1?

Reply:
This wasn't clear. I copied it from AWS and Azure code trusting that it works.